DoSomething/gambit
# Fetch askSubscriptionStatus replies from GraphQL

#### What's this PR do?

Removes the hardcoded macro replies for an `askSubscriptionStatus` broadcast, and refactors middleware to send replies from the GraphQL fields added in https://github.com/DoSomething/graphql/pull/49 and https://github.com/DoSomething/graphql/pull/51.

Also refactors `askSubscriptionStatus` broadcasts to no longer update a user's `sms_status` as `pending` upon receiving the broadcast, per [Slack thread](https://dosomething.slack.com/archives/C03T8SDMS/p1549318310005500). 

#### How should this be reviewed?

* Verify expected behavior for the various replies defined on an `askSubscriptionStatus` broadcast (`4qLXWRSqY0ommayY2iWCwU`) per the GraphQL `topic` query response --  see https://github.com/DoSomething/graphql/pull/51#issuecomment-460359256

Example:
<img width="500" alt="screen shot 2019-02-04 at 6 38 43 pm" src="https://user-images.githubusercontent.com/1236811/52250857-a6d66600-28ae-11e9-8775-eb82962c4cd3.png">

* Verify the `less` keyword sends the hardcoded text per https://github.com/DoSomething/gambit/pull/471#discussion_r253675066

Example:
<img width="500" alt="screen shot 2019-02-04 at 7 01 47 pm" src="https://user-images.githubusercontent.com/1236811/52250976-5d3a4b00-28af-11e9-9014-adc02dd5ef18.png">


#### Any background context you want to provide?

The last broadcast type left to port over to GraphQL is `askVotingPlanStatus`. Not that we're sending one of these any time soon, but it seems worth fixing it up vs scrapping the feature from the repo entirely.  

#### Relevant tickets
Fixes https://www.pivotaltracker.com/story/show/163095523

#### Checklist
